Rakefile

```diff
@@ -7,7 +7,7 @@ require "yard"
 Rake::TestTask.new(:test) do |t|
   t.libs << "test"
   t.libs << "lib"
-  t.test_files = FileList["test/**/*_test.rb"]
+  t.test_files = FileList[ENV["TESTS"] || "test/**/*_test.rb"]
```

Comment:
I'm pretty sure Minitest has this built in using the `TEST` env var
```sh
TEST=test/lib/**/*_test.rb bundle exec rake
```

Reply:
Ah, cool! I guess we can remove this from PVC then too, as I copied this line from there 😄 cc @khiga8